cd/bootstrap/cfn-templates/70-microservice_lambdas_deployer_pipeline.yaml

```diff
@@ -0,0 +1,799 @@
+# TODO: multiregion CFN deploy 
+
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Continuos Delivery pipeline for lambdas microservice deployment'
+Parameters:
+  ProjectName:
+    Description: Usually "pn", can be changed for feature branch deployment
+    Type: String
+  CodeStarGithubConnectionArnInfra:
+    Description: CodeStar connection for github infrastructure project
+    Type: String
+  CodeStarGithubConnectionArnMicro:
```

Comment:
Perché hai distinto le due connessioni? CodeStarGithubConnectionArnInfra e CodeStarGithubConnectionArnMicro

Reply:
Non è detto che i due repository siano raggiungibili dalla stessa connessione

Reply:
Ok, ma ora è così e non c'è in programma una modifica. Perché complicare più di quanto serve?